netstats/src/views/layout.jade

```diff
@@ -3,34 +3,55 @@ doctype html
 html(ng-app="netStatsApp")
   head
     meta(name="viewport", content="width=device-width, initial-scale=1.0, maximum-scale=1.0")
-    title Kowala Test Network
+    title Kowala - kUSD Test Network
     style(type="text/css") [ng\:cloak], [ng-cloak], [data-ng-cloak], [x-ng-cloak], .ng-cloak, .x-ng-cloak { display: none !important; }
-    link(rel='stylesheet', href='//fonts.googleapis.com/css?family=Montserrat:200,300,400,500,600"')
-    link(rel='stylesheet', href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css')
+    link(rel="stylesheet", href="https://fonts.googleapis.com/css?family=Poppins:300,400,500")
+    link(rel="stylesheet", href="https://cdn.kowala.tech/styles/kowala.css")
```

Comment:
This is always using the main public CDN. Is there a way to make it use the dev CDN for dev and the prod CDN for prod?

Reply:
Updated